### PR TITLE
durysta may be right for you section #48 and Text styling for columns blocks #36

### LINKED ISSUE #36: Text styling for columns blocks
Ensure that text styling in the columns blocks matches the source, e.g.

Screenshot 2023-06-30 at 14 43 04

image

### LINKED ISSUE #48: durysta may be right for you section
image

- [x] Font styling in the doc
- [x] alignment in css
- [x] grey background for Ask your doctor section and offer information
- [x] list items have a bit more separation (spacing) on live on all resolutions
- [x] The Ask Your Doctor section / grey box has some differences in the image and title area: image is larger, text column is narrower on live on tablet and desktop
- [x] Image is a bit larger on live on tablet landscape
- [x] layout inconsistencies coupled with font sizing differences on tablet portrait / mobile landscape
- [x] Image size and some spacing differences on mobile portrait
- [x] other styling observations

### PR DESCRIPTION
**Pull request by rootpea**
Fix #36
Fix #48

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page
- After: https://36-textstyling--durysta--hlxsites.hlx.page

### CODE REVIEW

**Comment by kmurugulla** [comment]
@rootpea - one small change that is needed is the grey box is not the full page size in the durysta.com 

image

**Review by kmurugulla — DISMISSED** [review]
the grey box is not full page size
image

File: styles/styles.css [hunk 1]
```diff
@@ -133,6 +136,7 @@
   --ff-FeltTipRomanW01-Reg: 'FeltTipRomanW01-Reg','FeltTipRomanW01-Reg-Fallback',sans-serif;
   --heading-font-family: sans-serif;
   --body-font-family: var(--ff-ITCAvantGardeGothicW01-Md);
+  --strong-font-family: var(--ff-ITCAvantGardeGothicW01-Dm);
```
Comment (raeloh) [inline styles/styles.css:139]: There are instances on https://www.durysta.com/ where bolded text has font family `'ITC Avant Garde Gothic W01 Bd'` (in the safety information section and brochure section for example). 

Will we make a decision as a team to introduce more consistency that all cases of bolded text should be styled the same?

Comment (rootpea) [reply styles/styles.css:139]: Happy to go with either Dm or Bd here
I'll open an issue to decide on which one we should use elsewhere.

Comment (rootpea) [reply styles/styles.css:139]: https://github.com/hlxsites/durysta/issues/55

**Comment by dragoss-nc** [comment]
Hello,

- On desktop (~1650px) the font-sizes don't match, the border between the 2 columns is missing and the top separator is thinner than on live:
image
- On tablet landscape same as on desktop:
image
- On tablet portrait the columns are still on top of each other in both sections, while on live they are side-by-side; also there are some font inconsistencies:
image
- On mobile portrait there are some font inconsistencies and the horizontal border is missing
image
- On mobile landscape the issues are similar to the ones for tablet portrait:
image

**Comment by rootpea** [comment]
tablet mode is addressed in https://github.com/hlxsites/durysta/issues/58
top separator has been fixed in https://github.com/hlxsites/durysta/issues/43

**Comment by rootpea** [comment]
I'll address the column/row separator in another PR

**Pull request merged**